content/guides/event-streaming/spring-cloud-stream-kafka-p1.md

```diff
@@ -294,4 +294,4 @@ The complete running code for this tutorial is available in [Github](https://git
 
 
 ## What’s Next
-In the next tutorial (Part 2), I’ll show a real life example such as the **PubSub** module, multiple Kafka topics and failover handling.
+In the next tutorial [Part 2](https://tanzu.vmware.com/developer/guides/event-streaming/spring-cloud-stream-kafka-p2/), I’ll show a real life example such as the **PubSub** module, multiple Kafka topics and failover handling.
```

Comment:
This link will not be available before this merge, probably I should add it in a separate merge.